Misc/NEWS.d/next/Library/2019-05-17-11-44-21.bpo-33524.8y_xUU.rst

```diff
@@ -0,0 +1,2 @@
+Fix the folding of email header when the max_line_length is 0 or None.
```

Comment:
To be clear about what we are fixing this probably ought to say "...is 0 or None and the header contains non-ascii characters".